packages/react-query/src/utils.ts

```diff
@@ -2,7 +2,7 @@ export function shouldThrowError<T extends (...args: any[]) => boolean>(
   _useErrorBoundary: boolean | T | undefined,
```

Comment:
What's the reason to prefix this `useErrorBoundary` arg with `_`? Should it be renamed to `_throwError` with `_` as well or `throwError` is fine?

Reply:
I think this function used to look differently and there was likely a name clash. `throwError` is fine